perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_user_attribute_def_def_login_namespace_fenix_persistent_shadow.java

```diff
@@ -41,7 +54,7 @@ public Attribute fillAttribute(PerunSessionImpl perunSession, User user, Attribu
 		Attribute filledAttribute = new Attribute(attribute);
 
 		if (attribute.getFriendlyName().equals(attrNameFenix)) {
-			filledAttribute.setValue(sha1HashCount(user, domainNameFenix).toString() + domainNameFenix);
+			filledAttribute.setValue(sha1HashCount(user, getDomainNameFenix()).toString() + getDomainNameFenix());
```

Comment:
I've just realized that the domain name property will be stored without the "@" character. However, we need to include the "@" while setting the attribute value.
So, please could you change the code to something like this?

```
if (attribute.getFriendlyName().equals(attrNameFenix)) {
    String domain = "@" + getDomainNameFenix();
    filledAttribute.setValue(sha1HashCount(user, domain).toString() + domain);
```

Reply:
I have applied your suggested changes.